pubsub_test.go

```diff
@@ -0,0 +1,101 @@
+package redigosrv
+
+import (
+	"context"
+	"errors"
+
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+)
+
+var _ = Describe("RedigoService (PubSub)", func() {
+	It("should subscribe and publish messages", func() {
```

Comment:
Make this test asynchronous. This way we will be able to ensure the timeout duration.

Reply:
Done :+1: